eternalcore-core/src/main/java/com/eternalcode/core/feature/playtime/PlaytimeCommand.java

```diff
@@ -0,0 +1,55 @@
+package com.eternalcode.core.feature.playtime;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.notice.NoticeService;
+import com.eternalcode.core.util.DurationUtil;
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+import org.bukkit.Statistic;
+import org.bukkit.entity.Player;
+
+import java.time.Duration;
+
+@Command(name = "playtime", aliases = {"play-time", "pt"})
```

Comment:
unnecessary and creates chaos
```suggestion
@Command(name = "playtime", aliases = {"pt"})
```

Reply:
I was inspired by the /elder-guardian and /end-screen commands you and Piotrulla created, but okay 😄